### ISSUE TITLE
WiderFace download issue on Windows, Linux and Colab - MD5 and Google Drive Virus scan warning. 

### ISSUE THREAD

**Issue by jayrmh** [pytorch/vision#8226]
### 🐛 Describe the bug

_I believe this issue should have been resolved before, but I cannot seem to circumvent this based on suggestions from previous threads. I have tried downloading manually and placing it in directory, skipping MD5 checks in utils.py, and seems to be an unhandled Google API issue. Please help,  code and errors below:_ 

``from torchvision.datasets import WIDERFace``
``from torch.utils.data import DataLoader``
``from torchvision import transforms``

``dataset_training = WIDERFace(root="C:\\Users\X\\WiderFaces", split="train", download=True)``

**ERROR BELOW:** 

> C:\ProgramData\Anaconda3\envs\framework\Lib\site-packages\torchvision\datasets\utils.py:260: UserWarning: We detected some HTML elements in the downloaded file. This most likely means that the download triggered an unhandled API response by GDrive. Please report this to torchvision at https://github.com/pytorch/vision/issues including the response:

<!DOCTYPE html><html><head><title>Google Drive - Virus scan warning</title><meta http-equiv="content-type" content="text/html; charset=utf-8"/><style nonce="Zi4saGbGx3Q7KqhzoB9CDw">.goog-link-button{position:relative;color:#15c;text-decoration:underline;cursor:pointer}.goog-link-button-disabled{color:#ccc;text-decoration:none;cursor:default}body{color:#222;font:normal 13px/1.4 arial,sans-serif;margin:0}.grecaptcha-badge{visibility:hidden}.uc-main{padding-top:50px;text-align:center}#uc-dl-icon{display:inline-block;margin-top:16px;padding-right:1em;vertical-align:top}#uc-text{display:inline-block;max-width:68ex;text-align:left}.uc-error-caption,.uc-warning-caption{color:#222;font-size:16px}#uc-download-link{text-decoration:none}.uc-name-size a{color:#15c;text-decoration:none}.uc-name-size a:visited{color:#61c;text-decoration:none}.uc-name-size a:active{color:#d14836;text-decoration:none}.uc-footer{color:#777;font-size:11px;padding-bottom:5ex;padding-top:5ex;text-align:center}.uc-footer a{color:#15c}.uc-footer a:visited{color:#61c}.uc-footer a:active{color:#d14836}.uc-footer-divider{color:#ccc;width:100%}.goog-inline-block{position:relative;display:-moz-inline-box;display:inline-block}* html .goog-inline-block{display:inline}*:first-child+html .goog-inline-block{display:inline}sentinel{}</style><link rel="icon" href="//ssl.gstatic.com/docs/doclist/images/drive_2022q3_32dp.png"/></head><body><div class="uc-main"><div id="uc-dl-icon" class="image-container"><div class="drive-sprite-aux-download-file"></div></div><div id="uc-text"><p class="uc-warning-caption">Google Drive can't scan this file for viruses.</p><p class="uc-warning-subcaption"><span class="uc-name-size"><a href="/open?id=15hGDLhsx8bLgLcIRD5DhYt5iBxnjNF1M">WIDER_train.zip</a> (1.4G)</span> is too large for Google to scan for viruses. Would you still like to download this file?</p><form id="download-form" action="https://drive.usercontent.google.com/download" method="get"><input type="submit" id="uc-download-link" class="goog-inline-block jfk-button jfk-button-action" value="Download anyway"/><input type="hidden" name="id" value="15hGDLhsx8bLgLcIRD5DhYt5iBxnjNF1M"><input type="hidden" name="export" value="download"><input type="hidden" name="confirm" value="t"><input type="hidden" name="uuid" value="b4a0a65f-a20a-4376-a637-c36bbb67b016"></form></div></div><div class="uc-footer"><hr class="uc-footer-divider"></div></body></html>
.
.
.
**The MD5 checksum of the download file **[C:\\Users\X\\WiderFaces\\WIDER_train.zip** does not match the one on record .Please delete the file and try again. If the issue persists, please report this to torchvision at https://github.com/pytorch/vision/issues.**

### Versions

```
PyTorch version: 2.1.2
Is debug build: False
CUDA used to build PyTorch: 11.8
ROCM used to build PyTorch: N/A

OS: Microsoft Windows 11 Home
GCC version: (GCC) 4.4.3
Clang version: Could not collect
CMake version: version 3.21.3
Libc version: N/A

Python version: 3.11.7 | packaged by conda-forge | (main, Dec 23 2023, 14:27:59) [MSC v.1937 64 bit (AMD64)] (64-bit runtime)
Python platform: Windows-10-10.0.22000-SP0
Is CUDA available: True
CUDA runtime version: 11.2.67
CUDA_MODULE_LOADING set to: LAZY
GPU models and configuration: GPU 0: NVIDIA GeForce RTX 3060 Laptop GPU
Nvidia driver version: 516.94
cuDNN version: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.2\bin\cudnn_ops_train64_8.dll
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

CPU:
Architecture=9
CurrentClockSpeed=3301
DeviceID=CPU0
Family=107
L2CacheSize=4096
L2CacheSpeed=
Manufacturer=AuthenticAMD
MaxClockSpeed=3301
Name=AMD Ryzen 9 5900HS with Radeon Graphics
ProcessorType=3
Revision=20480

Versions of relevant libraries:
[pip3] numpy==1.26.3
[pip3] torch==2.1.2
[pip3] torchaudio==2.1.2
[pip3] torchio==0.19.3
[pip3] torchvision==0.16.2
[conda] blas                      1.0                         mkl    conda-forge
[conda] mkl                       2023.2.0         h6a75c08_49573    conda-forge
[conda] mkl-service               2.4.0           py311h2bbff1b_1
[conda] mkl_fft                   1.3.8           py311h2bbff1b_0
[conda] mkl_random                1.2.4           py311h59b6b97_0
[conda] numpy                     1.26.3          py311hdab7c0b_0
[conda] numpy-base                1.26.3          py311hd01c5d8_0
[conda] pytorch                   2.1.2           py3.11_cuda11.8_cudnn8_0    pytorch
[conda] pytorch-cuda              11.8                 h24eeafa_5    pytorch
[conda] pytorch-mutex             1.0                        cuda    pytorch
[conda] torchaudio                2.1.2                    pypi_0    pypi
[conda] torchio                   0.19.3                   pypi_0    pypi
[conda] torchvision               0.16.2                   pypi_0    pypi
```

**Comment by NicolasHug** [comment]
@jayrmh thanks for the report.

> I have tried downloading manually and placing it in directory, skipping MD5 checks in utils.py, and seems to be an unhandled Google API issue

Just to make sure I understand: did manually downloading the files fix your issue?


FYI this issue has been reported a few times and is likely due to a change in googledrive APIs. We're investigating long-term solutions, probably adding `gdown` as an optional dependency
